akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/Running.scala

```diff
@@ -270,11 +223,9 @@ private[akka] object Running {
       msg match {
         case JournalResponse(r)                => onJournalResponse(r)
         case in: IncomingCommand[C @unchecked] => onCommand(in)
-        case SnapshotterResponse(r) =>
-          setup.log.warning("Unexpected SnapshotterResponse {}", r)
-          Behaviors.unhandled
-        case RecoveryTickEvent(_)  => Behaviors.unhandled
-        case RecoveryPermitGranted => Behaviors.unhandled
+        case SnapshotterResponse(r)            => onDeleteSnapshotResponse(r)
```

Comment:
Does the snapshotter only ever respond on delete and reach here, no chance we could get a save-ok?